scene/main/scene_tree.cpp

```diff
@@ -1330,6 +1310,25 @@ void SceneTree::add_current_scene(Node *p_current) {
 }
 #ifdef DEBUG_ENABLED
 
+static void _fill_array(Node *p_node, Array &array, int p_level) {
```

Comment:
You can probably move all this back to the debugger so there's less clutter in SceneTree